code/modules/client/preference/preferences.dm

```diff
@@ -197,7 +197,7 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 			dat += "<h2>Identity</h2>"
 			dat += "<b>Gender:</b> <a href='?_src_=prefs;preference=gender'>[active_character.gender == MALE ? "Male" : (active_character.gender == FEMALE ? "Female" : "Genderless")]</a>"
 			dat += "<br>"
-			dat += "<b>Age:</b> <a href='?_src_=prefs;preference=age;task=input'>[active_character.age]</a><br>"
+			dat += "<b>Age:</b> <a href='?_src_=prefs;preference=age;task=input'>[clamp(active_character.age, S.min_age, S.max_age)]</a><br>"
```

Comment:
this is the incorrect place to clamp this value, should instead be clamping the input when it's sent to `topic()`, that way we make sure we're ensuring the displayed value is lining up with the actual value of the variable.

Reply:
This place is better?